packages/core/src/pill-input/PillInput.css

```diff
@@ -282,21 +282,48 @@
   display: contents;
 }
 
+.saltPillInput .saltPill {
```

Comment:
These changes should be done in Pill.css and only the overrides needed for PillInput should be added here